### PR TITLE
WIP - Basic support for serving admin pages

### LINKED ISSUE #13: Add support for admin pages
Each service should be able to provide resources such as HTML, CSS, etc, which Misk can combine to form an admin dashboard. The data displayed would be used for debugging and administrative purposes, such as setting dynamic config.

### PR DESCRIPTION
**Pull request by JGulbronson**
Allows serving static css, html, js files located at resources/misk/admin without caching.

Closes #13

### CODE REVIEW

File: misk/src/main/resources/misk/admin/html/home.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```
Comment (mmihic) [inline misk/src/main/resources/misk/admin/html/home.html]: We almost definitely need to allow this to be customized by the application.

We also likely need some sort of templating support for HTML content. That could be a separate PR with some sort of resource resolver that takes the name of an admin resource and a data class/bean/map containing the template parameters, and writes to an output stream (i.e. a library class not an action)

Comment (JGulbronson) [reply misk/src/main/resources/misk/admin/html/home.html]: Yep, I was looking at templating engines. I think with this I mostly want to get the serving of static files set up, and then focus on building out the admin side of things.
